crate/client/src/kms_rest_client.rs

```diff
@@ -503,12 +505,8 @@ impl KmsClient {
         // Build the client
         Ok(Self {
             client: builder
-                .connect_timeout(Duration::from_secs(10))
-                .timeout(Duration::from_secs(10))
-                .tcp_keepalive(Duration::from_secs(5))
-                .pool_idle_timeout(Duration::from_secs(5))
-                .pool_max_idle_per_host(2)
                 .default_headers(headers)
+                .tcp_keepalive(Duration::from_secs(60))
```

Comment:
The problem [hyper](https://github.com/hyperium/hyper/issues/2136) is occurring (from time to time) when client authentication by certificate is enabled on the Rust Client.
I believe the 60s delay has no impact on this problem. 
Currently, I've disabled the certificate authentication wherever it is not needed (this is only a workaround). It could be fixed properly using reqwest and rustls-tls instead of native-tls.